src/main/java/com/github/galatynf/sihywtcamd/mixin/zombie/ZombieLivingMixin.java

```diff
@@ -91,6 +91,7 @@ private void applyDifferentAttributes(float chanceMultiplier, CallbackInfo ci) {
                 double value = 0.5 + 1.0 * chanceMultiplier + this.random.nextDouble();
                 this.getAttributeInstance(EntityAttributes.MAX_HEALTH).addPersistentModifier(
                         new EntityAttributeModifier(TANK_BONUS_ID, value, EntityAttributeModifier.Operation.ADD_MULTIPLIED_BASE));
+                this.setHealth((float) (20 + 20 * value));
```

Comment:
Oh didn't know that was needed? When I added that it worked without it... if it's needed, then you should call `this.setHealth(this.getMaxHealth());` (or something like that, I'm not on the code rn)